freex/src/components/Auth/Login/Login.module.css

```diff
@@ -1,37 +1,50 @@
 .container {
   display: grid;
-  margin-top: 6rem;
   grid-template-areas: "left right";
-  width: 100%;
+  grid-template-columns: 1fr 1fr;
+  padding: 8rem 2rem 2rem 0;
```

Comment:
Masz bardzo duży padding-top - 8 rem - ja bym dała mniej.  Przez to jest taka duża odległośc pomiędzy nawigacją i treścią strony i trzeba więcej scrollować